### PR TITLE
Fixing duplicated alerts descriptions

### PR DESCRIPTION
**Pull request by DProvinciani**
This pull request modifies the string displayed in the `description` field of the `vulnerability-detector` alerts. The main purpose id to avoid a duplicated data with the `rationale` or `title` field of the alert.

Here is an example of how an alert looks now:

```json
{
    "timestamp": "2020-04-30T20:51:55.526-0300",
    "rule": {
        "level": 10,
        "description": "CVE-2016-6489 affects libhogweed4.",
        "id": "23505",
        "firedtimes": 1122,
        "mail": false,
        "groups": [
            "vulnerability-detector"
        ],
        "gdpr": [
            "IV_35.7.d"
        ],
        "pci_dss": [
            "11.2.1",
            "11.2.3"
        ]
    },
    "agent": {
        "id": "001",
        "name": "ubuntu18-wagent",
        "ip": "192.168.0.124"
    },
    "manager": {
        "name": "centos8-wserver"
    },
    "id": "1588290715.8232793",
    "decoder": {
        "name": "json"
    },
    "data": {
        "vulnerability": {
            "cve": "CVE-2016-6489",
            "title": "The RSA and DSA decryption code in Nettle makes it easier for attackers to discover private keys via a cache side channel attack.",
            "severity": "High",
            "alert_origin": "NVD feed",
            "published": "2017-04-14",
            "updated": "2017-07-01",
            "state": "Pending confirmation",
            "cvss": {
                "cvss2": {
                    "vector": {
                        "attack_vector": "network",
                        "access_complexity": "low",
                        "authentication": "none",
                        "confidentiality_impact": "low",
                        "integrity_impact": "none",
                        "availability": "none"
                    },
                    "base_score": "5"
                },
                "cvss3": {
                    "vector": {
                        "attack_vector": "network",
                        "access_complexity": "low",
                        "confidentiality_impact": "low",
                        "integrity_impact": "none",
                        "availability": "none",
                        "privileges_required": "none",
                        "user_interaction": "none",
                        "scope": "unchanged"
                    },
                    "base_score": "7.500000"
                }
            },
            "package": {
                "name": "libhogweed4",
                "source": "nettle",
                "version": "3.4-1",
                "architecture": "amd64",
                "condition": "CVE-2016-6489 reports as vulnerable all the versions of this package"
            },
            "cwe_reference": "CWE-310",
            "references": [
                "http://rhn.redhat.com/errata/RHSA-2016-2582.html",
                "http://www.openwall.com/lists/oss-security/2016/07/29/7",
                "http://www.ubuntu.com/usn/USN-3193-1",
                "https://bugzilla.redhat.com/show_bug.cgi?id=1362016",
                "https://eprint.iacr.org/2016/596.pdf",
                "https://git.lysator.liu.se/nettle/nettle/commit/3fe1d6549765ecfb24f0b80b2ed086fdc818bff3",
                "https://security.gentoo.org/glsa/201706-21",
                "https://www.oracle.com/security-alerts/cpuapr2020.html"
            ],
            "assigner": "cve@mitre.org",
            "cve_version": "4.0"
        }
    },
    "location": "vulnerability-detector"
}
```

### CODE REVIEW

File: rules/0520-vulnerability-detector_rules.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
       <if_sid>23501</if_sid>
       <options>no_full_log</options>
       <field name="vulnerability.severity">High</field>
-      <description>$(vulnerability.title)</description>
+      <description>$(vulnerability.cve) affects $(vulnerability.package.name).</description>
```
Comment (chemamartinez) [inline rules/0520-vulnerability-detector_rules.xml:49]: Remove the dot at the end of the description

File: rules/0520-vulnerability-detector_rules.xml [hunk 2]
```diff
@@ -60,7 +60,7 @@
       <if_sid>23501</if_sid>
       <options>no_full_log</options>
       <field name="vulnerability.severity">Critical</field>
-      <description>$(vulnerability.title)</description>
+      <description>$(vulnerability.cve) affects $(vulnerability.package.name).</description>
```
Comment (chemamartinez) [inline rules/0520-vulnerability-detector_rules.xml:63]: Remove the dot at the end of the description. Please do the same for all the rules of the file.

File: rules/0520-vulnerability-detector_rules.xml (hunk 1 above)
Comment (DProvinciani) [reply rules/0520-vulnerability-detector_rules.xml]: DONE

File: rules/0520-vulnerability-detector_rules.xml (hunk 2 above)
Comment (DProvinciani) [reply rules/0520-vulnerability-detector_rules.xml]: DONE

**Comment by DProvinciani** [comment]
Closing because of commits conflicts.

**Comment by DProvinciani** [comment]
These changes are now added in [pull 638](https://github.com/wazuh/wazuh-ruleset/pull/638).

**Pull request closed without merging**